modules/iam/outputs.tf

```diff
@@ -1,6 +1,6 @@
 output "config" {
   value = {
-    service_role = aws_iam_role.eks_service_role.name
+    cluster_role = aws_iam_role.eks_cluster_role.name
```

Comment:
Should the eks_service_policy be removed at L10, given that it's been removed as a resource ?

Reply:
No, I think we need this line. 

The IAM role for an EKS control plane (a.k.a master or cluster) is still necessary with `AmazonEKSClusterPolicy` but the `AmazonEKSServicePolicy` is no more necessary. For that reason, I'd like to change the name of the IAM role.

Reply:
@takanabe Line 10, `aws_iam_role_policy_attachment.eks_service_policy` but you've removed it above.